README.md

```diff
@@ -1 +1,169 @@
-# hello-world-frontend
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] Remove this header link if you have no live demo link and section.

Reply:
LOL